Formula/grpc.rb

```diff
@@ -32,9 +31,23 @@ class Grpc < Formula
   depends_on "protobuf"
   depends_on "re2"
 
+  uses_from_macos "llvm" => [:build, :test], since: :catalina
   uses_from_macos "zlib"
 
+  fails_with :clang do
+    build 1100
+  end
+
+  fails_with gcc: "5"
+  fails_with gcc: "6"
+  fails_with gcc: "7"
+  fails_with gcc: "8"
+  fails_with gcc: "9"
+  fails_with gcc: "10"
+  fails_with :gcc
```

Comment:
```suggestion
  fails_with :gcc
```

But this isn't really true. GCC works absolutely fine on Linux at least.